src/hooks/useClickOutside.ts

```diff
@@ -24,18 +20,18 @@ export const useClickOutside = <E extends HTMLElement>(
         !isValidTarget ||
         (wrapperRef.current &&
           !wrapperRef.current?.contains(target) &&
-          target.id !== id)
+          wrapperRef.current !== target)
       ) {
         onClickOutside();
       }
     },
-    [id, onClickOutside]
+    [onClickOutside]
   );
 
   useEffect(() => {
     document.addEventListener("click", handleClickOutside, false);
     return () =>
       document.removeEventListener("click", handleClickOutside, false);
   }, [handleClickOutside]);
-  return [wrapperRef, id];
+  return [wrapperRef];
```

Comment:
This might as well just return `wrapperRef` without the array.

Reply:
True, I'll update.